File: solutions/ruby/raindrops/1/raindrops.rb
module Raindrops
  def self.convert(number)
    final = ''

    if number % 3 == 0
      final << 'Pling'
    end
    if number % 5 == 0
      final << 'Plang'
    end
    if number % 7 == 0
      final << 'Plong'
    end

    final != '' ? final : number.to_s
  end
end

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "strconv"

func Convert(number int) string {
    name := ""
    
	if number%3 == 0 {
        name += "Pling"
    }
    if number%5 == 0 {
        name += "Plang"
    }
    if number%7 == 0 {
        name += "Plong"
    }
    if name == "" {
        return strconv.Itoa(number)
    }
    return name
}
